src/components/AdditionalInfo/AdditionalInfo.jsx

```diff
@@ -69,4 +82,18 @@ AdditionalInfo.propTypes = {
    */
   triggerText: PropTypes.string.isRequired,
   onClick: PropTypes.func,
+  /**
+   * Child elements (content) of modal when displayed
+   */
+  children: PropTypes.node,
+  /**
+   * Function to pass in events and parameters to the dataLayer
+   * and (eventually) into Google Analytics
+   */
+  trackEvent: PropTypes.func,
```

Comment:
Where is this used? I don't see any call to a `trackEvent` function..

Reply:
Nope - an artifact of the first version of this.